a = 4

assert(a<5)
print("断言成功，程序继续")

assert(a>4)  # 断言失败，程序报错


#  所以assert常常被用作调试代码使用